pkg/install/images/images.go

```diff
@@ -328,23 +328,30 @@ func LoadImages(ctx context.Context, log logrus.FieldLogger, archivePath string,
 	return nil
 }
 
-func CopyImages(ctx context.Context, log logrus.FieldLogger, dryRun bool, images []string, registry string, userAgent string) (int, int, error) {
+func CopyImages(ctx context.Context, log logrus.FieldLogger, dryRun bool, images []string, registry string, userAgent string) (int, int, []string, error) {
```

Comment:
Instead of []string, this should still simply return an error with all the failed images included, if any.